presidio-api/cmd/presidio-api/actions.go

```diff
@@ -74,6 +77,108 @@ func (api *API) anonymize(c *gin.Context) {
 	}
 }
 
+func (api *API) anonymizeImage(c *gin.Context) {
+	var anonymizeImageAPIRequest types.AnonymizeImageApiRequest
+
+	project := c.Param("project")
+
+	anonymizeImageAPIRequest.ImageType = c.PostForm("imageType")
+	if anonymizeImageAPIRequest.ImageType == "" {
+		c.AbortWithError(http.StatusBadRequest, fmt.Errorf("Image type is missing (image/jpeg, image/png, image/tiff, image/gif, image/bmp)"))
```

Comment:
you need to merge with my changes.. it should be server.AbortWithError...

Reply:
Done